clouddrift/ragged.py

```diff
@@ -787,9 +790,9 @@ def subset(
 
 
 def unpack(
-    ragged_array: np.ndarray,
-    rowsize: np.ndarray[int],
-    rows: int | Iterable[int] = None,
+    ragged_array: np.ndarray | xr.DataArray,
+    rowsize: np.ndarray[int] | xr.DataArray,
+    rows: int | np.int_ | Iterable[int] | None = None,
```

Comment:
is it necessary to have both `int` and `np.int_?`

Reply:
I can check on this